Packs/Slack/Integrations/SlackEventCollector/SlackEventCollector.yml

```diff
@@ -0,0 +1,112 @@
+category: Analytics & SIEM
+commonfields:
+  id: Slack Event Collector
+  version: -1
+configuration:
+- display: ''
+  displaypassword: User Token
+  name: user_token
+  type: 9
+  required: false
+  hiddenusername: true
+- defaultvalue: slack
+  display: The product name corresponding to the integration that originated the events
+  name: product
+  required: false
+  type: 0
+- defaultvalue: slack
+  display: The vendor name corresponding to the integration that originated the events
+  name: vendor
+  required: false
+  type: 0
+- display: The maximum number of audit logs to fetch
+  name: limit
+  type: 0
+  defaultvalue: 1000
+  required: false
+- display: First fetch time interval
+  name: oldest
```

Comment:
`prepare_query_params()` is parsing this param, but is used for the corresponding arguments in `slack-get-events` as well, and I wanted to keep the naming of the arguments like the API. do you think I should create another version of this function for args because of the naming?

Reply:
Obviously different APIs use different namings but in general, I always recommend naming arguments meaningfully and maintaining some level of alignment between similar arguments in different integrations, this makes more sense to customers when working in XSOAR. In your case since we don't really have commands if it makes a lot of mess in your code then let's drop it.